packages/react-core/src/components/Toolbar/examples/Toolbar.md

```diff
@@ -239,6 +239,66 @@ class ToolbarSpacers extends React.Component {
 }
 ```
 
+### Sticky
+
+```js
+import React from 'react';
+import { Toolbar, ToolbarItem, ToolbarContent } from '@patternfly/react-core';
+import { Button, ButtonVariant, InputGroup, TextInput, Checkbox } from '@patternfly/react-core';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+
+const ToolbarItems = () => {
+  const [isSticky, setIsSticky] = React.useState(true);
+  const [showEvenOnly, setShowEvenOnly] = React.useState(true);
+
+  return (
+    <React.Fragment>
+      <Checkbox
+        label="Is toolbar sticky"
+        isChecked={isSticky}
+        onChange={setIsSticky}
+        aria-label="checkbox for is sticky"
+        id="isStickyCheckbox"
+      />
+      <div style={{ overflowY: 'scroll', height: '200px' }}>
+        <Toolbar
+          id="toolbar-spacers"
+          inset={{default: 'insetNone'}}
+          isSticky={isSticky}
+        >
+          <ToolbarContent>
+            <ToolbarItem>
+              <InputGroup>
+                <TextInput name="textInput1" id="textInput1" type="search" aria-label="search input example" />
+                <Button variant={ButtonVariant.control} aria-label="search button for search input">
+                  <SearchIcon />
+                </Button>
+              </InputGroup>
+            </ToolbarItem>
+            <ToolbarItem>
+              <Checkbox
+                label="Show only even number items"
+                isChecked={showEvenOnly}
+                onChange={setShowEvenOnly}
+                aria-label="checkbox for showing only even numbers"
+                id="showOnlyEvenCheckbox"
+              />
+            </ToolbarItem>
+          </ToolbarContent>
+        </Toolbar>
+        <ul>
+          {[...Array(25).keys()]
+            .filter(number => (showEvenOnly ? number % 2 === 0 : number || number === 0))
```

Comment:
I think this would be cleaner if we moved the array creation and filtering out of the jsx, and conditionally ran the filter rather than running the filter regardless of if `showEvenOnly` is true.

Reply:
Agree! thanks, I changed it